assets/js/mixins/formatter.js

```diff
@@ -33,23 +41,32 @@ export default {
       val = Math.abs(val);
       return val >= this.fmtLimit ? this.round(val / 1e3, this.fmtDigits) : this.round(val, 0);
     },
-    fmtKw: function (watt = 0, kw = true, withUnit = true, digits) {
-      if (digits === undefined) {
-        digits = kw ? 1 : 0;
+    fmtW: function (watt = 0, format = WATT_FORMAT.KW, withUnit = true, digits) {
+      if (WATT_FORMAT.AUTO === format) {
+        if (watt >= 1_000_000) {
+          format = WATT_FORMAT.MW;
```

Comment:
You're modifying input params here. Didn't eslint complain? Should introduce a new variable containing the final format to use.

Reply:
I didn't want to insert any new variables. This makes the function look slimmer and more elegant.

Running `npm run lint` doesn't output warnings or errors:
```txt
npm run lint

> lint
> prettier assets './**/*.{yml,yaml}' --write && eslint assets/**/*.js assets/**/*.vue --fix --max-warnings=0

assets/css/app.css 95ms (unchanged)
assets/histoire.setup.js 55ms (unchanged)
assets/index.html 48ms (unchanged)
assets/js/api.js 27ms (unchanged)
assets/js/app.js 18ms (unchanged)
assets/js/auth.js 20ms (unchanged)
assets/js/baseapi.js 8ms (unchanged)
assets/js/co2Reference.js 6ms (unchanged)
assets/js/communityApi.js 4ms (unchanged)
assets/js/components/AnimatedNumber.vue 13ms (unchanged)
assets/js/components/BatterySettingsModal.vue 165ms (unchanged)
assets/js/components/ChargingPlan.story.vue 18ms (unchanged)
assets/js/components/ChargingPlan.vue 57ms (unchanged)
assets/js/components/ChargingPlanArrival.vue 37ms (unchanged)
assets/js/components/ChargingPlanPreview.story.vue 22ms (unchanged)
assets/js/components/ChargingPlanPreview.test.js 18ms (unchanged)
assets/js/components/ChargingPlanPreview.vue 43ms (unchanged)
assets/js/components/ChargingPlanSettings.vue 39ms (unchanged)
assets/js/components/ChargingPlanSettingsEntry.vue 64ms (unchanged)
assets/js/components/ChargingPlanWarnings.vue 23ms (unchanged)
assets/js/components/ChargingSessionModal.vue 44ms (unchanged)
assets/js/components/Config/AddDeviceButton.vue 6ms (unchanged)
assets/js/components/Config/CircuitsModal.vue 6ms (unchanged)
assets/js/components/Config/ControlModal.vue 24ms (unchanged)
assets/js/components/Config/defaultYaml/circuits.yaml 18ms (unchanged)
assets/js/components/Config/defaultYaml/eebus.yaml 1ms (unchanged)
assets/js/components/Config/defaultYaml/hems.yaml 1ms (unchanged)
assets/js/components/Config/defaultYaml/messaging.yaml 3ms (unchanged)
assets/js/components/Config/defaultYaml/modbusproxy.yaml 2ms (unchanged)
assets/js/components/Config/defaultYaml/tariffs.yaml 2ms (unchanged)
assets/js/components/Config/DeviceCard.vue 11ms (unchanged)
assets/js/components/Config/DeviceTags.vue 22ms (unchanged)
assets/js/components/Config/EebusModal.vue 5ms (unchanged)
assets/js/components/Config/FormRow.vue 16ms (unchanged)
assets/js/components/Config/GeneralConfig.vue 40ms (unchanged)
assets/js/components/Config/HemsModal.vue 9ms (unchanged)
assets/js/components/Config/InfluxModal.vue 23ms (unchanged)
assets/js/components/Config/JsonModal.vue 27ms (unchanged)
assets/js/components/Config/MessagingModal.vue 8ms (unchanged)
assets/js/components/Config/MeterModal.vue 62ms (unchanged)
assets/js/components/Config/mixins/test.js 4ms (unchanged)
assets/js/components/Config/Modbus.vue 98ms (unchanged)
assets/js/components/Config/ModbusProxyModal.vue 5ms (unchanged)
assets/js/components/Config/MqttModal.vue 19ms (unchanged)
assets/js/components/Config/NetworkModal.vue 11ms (unchanged)
assets/js/components/Config/PropertyCollapsible.vue 11ms (unchanged)
assets/js/components/Config/PropertyEntry.vue 9ms (unchanged)
assets/js/components/Config/PropertyField.vue 44ms (unchanged)
assets/js/components/Config/SponsorModal.vue 16ms (unchanged)
assets/js/components/Config/TariffsModal.vue 5ms (unchanged)
assets/js/components/Config/TestResult.vue 14ms (unchanged)
assets/js/components/Config/TitleModal.vue 8ms (unchanged)
assets/js/components/Config/UserInterfaceSettings.vue 24ms (unchanged)
assets/js/components/Config/VehicleModal.vue 67ms (unchanged)
assets/js/components/Config/YamlEditor.vue 15ms (unchanged)
assets/js/components/Config/YamlModal.vue 25ms (unchanged)
assets/js/components/CustomSelect.vue 11ms (unchanged)
assets/js/components/Energyflow/BatteryIcon.story.vue 7ms (unchanged)
assets/js/components/Energyflow/BatteryIcon.vue 4ms (unchanged)
assets/js/components/Energyflow/Energyflow.story.vue 38ms (unchanged)
assets/js/components/Energyflow/Energyflow.vue 66ms (unchanged)
assets/js/components/Energyflow/EnergyflowEntry.vue 19ms (unchanged)
assets/js/components/Energyflow/index.js 1ms (unchanged)
assets/js/components/Energyflow/LabelBar.vue 9ms (unchanged)
assets/js/components/Energyflow/Visualization.vue 32ms (unchanged)
assets/js/components/Footer.story.vue 5ms (unchanged)
assets/js/components/Footer.vue 4ms (unchanged)
assets/js/components/FormRow.vue 6ms (unchanged)
assets/js/components/GenericModal.vue 7ms (unchanged)
assets/js/components/GlobalSettingsModal.vue 4ms (unchanged)
assets/js/components/HelpModal.vue 17ms (unchanged)
assets/js/components/LabelAndValue.vue 10ms (unchanged)
assets/js/components/LimitEnergySelect.vue 14ms (unchanged)
assets/js/components/LimitSocSelect.vue 15ms (unchanged)
assets/js/components/LiveCommunity.story.vue 2ms (unchanged)
assets/js/components/LiveCommunity.vue 18ms (unchanged)
assets/js/components/Loadpoint.story.vue 10ms (unchanged)
assets/js/components/Loadpoint.vue 38ms (unchanged)
assets/js/components/Loadpoints.story.vue 13ms (unchanged)
assets/js/components/Loadpoints.vue 28ms (unchanged)
assets/js/components/LoadpointSessionInfo.vue 21ms (unchanged)
assets/js/components/LoadpointSettingsButton.vue 4ms (unchanged)
assets/js/components/LoadpointSettingsModal.vue 54ms (unchanged)
assets/js/components/LoginModal.vue 33ms (unchanged)
assets/js/components/Logo.vue 4ms (unchanged)
assets/js/components/MaterialIcon/Circuits.vue 3ms (unchanged)
assets/js/components/MaterialIcon/Climater.vue 3ms (unchanged)
assets/js/components/MaterialIcon/CloudOffline.vue 2ms (unchanged)
assets/js/components/MaterialIcon/Dropdown.vue 3ms (unchanged)
assets/js/components/MaterialIcon/DynamicPrice.vue 3ms (unchanged)
assets/js/components/MaterialIcon/Edit.vue 2ms (unchanged)
assets/js/components/MaterialIcon/Eebus.vue 4ms (unchanged)
assets/js/components/MaterialIcon/Hems.vue 2ms (unchanged)
assets/js/components/MaterialIcon/Influx.vue 3ms (unchanged)
assets/js/components/MaterialIcon/ModbusProxy.vue 4ms (unchanged)
assets/js/components/MaterialIcon/Mqtt.vue 3ms (unchanged)
assets/js/components/MaterialIcon/Notification.vue 3ms (unchanged)
assets/js/components/MaterialIcon/PlanEnd.vue 3ms (unchanged)
assets/js/components/MaterialIcon/PlanStart.vue 3ms (unchanged)
assets/js/components/MaterialIcon/Play.vue 3ms (unchanged)
assets/js/components/MaterialIcon/Reconnect.vue 4ms (unchanged)
assets/js/components/MaterialIcon/Record.vue 3ms (unchanged)
assets/js/components/MaterialIcon/Restart.vue 3ms (unchanged)
assets/js/components/MaterialIcon/RfidWait.vue 4ms (unchanged)
assets/js/components/MaterialIcon/SunDown.vue 4ms (unchanged)
assets/js/components/MaterialIcon/SunUp.vue 4ms (unchanged)
assets/js/components/MaterialIcon/Sync.vue 3ms (unchanged)
assets/js/components/MaterialIcon/VehicleLimit.vue 2ms (unchanged)
assets/js/components/MaterialIcon/VehicleLimitReached.vue 4ms (unchanged)
assets/js/components/MaterialIcon/VehicleLimitWarning.vue 3ms (unchanged)
assets/js/components/MaterialIcon/VehicleMinSoc.vue 4ms (unchanged)
assets/js/components/MaterialIcon/Welcome.vue 4ms (unchanged)
assets/js/components/Mode.story.vue 4ms (unchanged)
assets/js/components/Mode.vue 10ms (unchanged)
assets/js/components/MultiIcon/1.vue 1ms (unchanged)
assets/js/components/MultiIcon/2.vue 2ms (unchanged)
assets/js/components/MultiIcon/3.vue 2ms (unchanged)
assets/js/components/MultiIcon/4.vue 2ms (unchanged)
assets/js/components/MultiIcon/5.vue 2ms (unchanged)
assets/js/components/MultiIcon/6.vue 2ms (unchanged)
assets/js/components/MultiIcon/7.vue 2ms (unchanged)
assets/js/components/MultiIcon/8.vue 2ms (unchanged)
assets/js/components/MultiIcon/9.vue 2ms (unchanged)
assets/js/components/MultiIcon/index.js 1ms (unchanged)
assets/js/components/MultiIcon/MultiIcon.story.vue 5ms (unchanged)
assets/js/components/MultiIcon/MultiIcon.vue 6ms (unchanged)
assets/js/components/MultiIcon/Plus.vue 2ms (unchanged)
assets/js/components/MultiSelect.vue 16ms (unchanged)
assets/js/components/Notifications.story.vue 6ms (unchanged)
assets/js/components/Notifications.vue 15ms (unchanged)
assets/js/components/OfflineIndicator.story.vue 2ms (unchanged)
assets/js/components/OfflineIndicator.vue 14ms (unchanged)
assets/js/components/PasswordModal.vue 18ms (unchanged)
assets/js/components/Phases.story.vue 13ms (unchanged)
assets/js/components/Phases.vue 11ms (unchanged)
assets/js/components/Savings.story.vue 4ms (unchanged)
assets/js/components/Savings.vue 39ms (unchanged)
assets/js/components/SavingsTile.story.vue 1ms (unchanged)
assets/js/components/SavingsTile.vue 13ms (unchanged)
assets/js/components/SelectGroup.story.vue 6ms (unchanged)
assets/js/components/SelectGroup.vue 8ms (unchanged)
assets/js/components/Site.vue 18ms (unchanged)
assets/js/components/SmartCostLimit.vue 35ms (unchanged)
assets/js/components/Sponsor.story.vue 3ms (unchanged)
assets/js/components/Sponsor.vue 16ms (unchanged)
assets/js/components/SponsorTokenExpires.story.vue 3ms (unchanged)
assets/js/components/SponsorTokenExpires.vue 5ms (unchanged)
assets/js/components/TariffChart.vue 18ms (unchanged)
assets/js/components/TelemetrySettings.vue 12ms (unchanged)
assets/js/components/TopHeader.vue 7ms (unchanged)
assets/js/components/TopNavigation.story.vue 5ms (unchanged)
assets/js/components/TopNavigation.vue 25ms (unchanged)
assets/js/components/Vehicle.story.vue 18ms (unchanged)
assets/js/components/Vehicle.vue 26ms (unchanged)
assets/js/components/VehicleIcon/Airpurifier.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Bike.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Bus.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Climate.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Compute.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Cooler.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Device.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Dishwasher.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Generic.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Heater.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Heatpump.vue 1ms (unchanged)
assets/js/components/VehicleIcon/index.js 1ms (unchanged)
assets/js/components/VehicleIcon/Laundry.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Meter.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Moped.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Motorcycle.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Scooter.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Shuttle.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Tool.vue 1ms (unchanged)
assets/js/components/VehicleIcon/Tractor.vue 2ms (unchanged)
assets/js/components/VehicleIcon/Van.vue 1ms (unchanged)
assets/js/components/VehicleIcon/VehicleIcon.story.vue 6ms (unchanged)
assets/js/components/VehicleIcon/VehicleIcon.vue 8ms (unchanged)
assets/js/components/VehicleIcon/WaterHeater.vue 1ms (unchanged)
assets/js/components/VehicleOptions.vue 9ms (unchanged)
assets/js/components/VehicleSoc.vue 33ms (unchanged)
assets/js/components/VehicleStatus.story.vue 30ms (unchanged)
assets/js/components/VehicleStatus.test.js 11ms (unchanged)
assets/js/components/VehicleStatus.vue 42ms (unchanged)
assets/js/components/VehicleTitle.vue 13ms (unchanged)
assets/js/components/Version.story.vue 7ms (unchanged)
assets/js/components/Version.vue 25ms (unchanged)
assets/js/experimental.js 2ms (unchanged)
assets/js/featureflags.js 2ms (unchanged)
assets/js/i18n.js 7ms (unchanged)
assets/js/mixins/breakpoint.js 2ms (unchanged)
assets/js/mixins/collector.js 2ms (unchanged)
assets/js/mixins/formatter.js 21ms (unchanged)
assets/js/mixins/formatter.test.js 15ms (unchanged)
assets/js/mixins/icon.js 2ms (unchanged)
assets/js/restart.js 2ms (unchanged)
assets/js/router.js 4ms (unchanged)
assets/js/sessionInfo.js 2ms (unchanged)
assets/js/settings.js 6ms (unchanged)
assets/js/store.js 3ms (unchanged)
assets/js/theme.js 4ms (unchanged)
assets/js/units.js 2ms (unchanged)
assets/js/utils/deepEqual.js 1ms (unchanged)
assets/js/utils/energyOptions.js 4ms (unchanged)
assets/js/utils/fatal.js 1ms (unchanged)
assets/js/utils/linkify.js 1ms (unchanged)
assets/js/utils/linkify.test.js 1ms (unchanged)
assets/js/utils/native.js 1ms (unchanged)
assets/js/utils/smartCostAvailable.js 1ms (unchanged)
assets/js/views/App.vue 14ms (unchanged)
assets/js/views/ChargingSessions.vue 68ms (unchanged)
assets/js/views/Config.vue 84ms (unchanged)
assets/js/views/Log.vue 37ms (unchanged)
assets/js/views/Main.vue 3ms (unchanged)
assets/public/meta/site.webmanifest 3ms (unchanged)
.github/dependabot.yml 8ms (unchanged)
.github/FUNDING.yml 4ms (unchanged)
.github/ISSUE_TEMPLATE/config.yml 2ms (unchanged)
.github/workflows/default.yml 23ms (unchanged)
.github/workflows/docs-issue.yml 5ms (unchanged)
.github/workflows/documentation.yml 5ms (unchanged)
.github/workflows/nightly.yml 15ms (unchanged)
.github/workflows/release.yml 17ms (unchanged)
.github/workflows/schema.yml 4ms (unchanged)
.github/workflows/website.yml 4ms (unchanged)
.golangci.yml 3ms (unchanged)
.goreleaser-nightly.yml 8ms (unchanged)
.goreleaser.yml 10ms (unchanged)
evcc.dist.yaml 15ms (unchanged)
templates/definition/charger/abb.yaml 2ms (unchanged)
templates/definition/charger/abl-em4.yaml 3ms (unchanged)
templates/definition/charger/abl.yaml 3ms (unchanged)
templates/definition/charger/ac-elwa-2.yaml 2ms (unchanged)
templates/definition/charger/alfen.yaml 4ms (unchanged)
templates/definition/charger/alphatec.yaml 4ms (unchanged)
templates/definition/charger/amperfied-solar.yaml 2ms (unchanged)
templates/definition/charger/amperfied.yaml 2ms (unchanged)
templates/definition/charger/bender.yaml 6ms (unchanged)
templates/definition/charger/cfos.yaml 3ms (unchanged)
templates/definition/charger/dadapower.yaml 2ms (unchanged)
templates/definition/charger/daheimladen-mb.yaml 3ms (unchanged)
templates/definition/charger/daheimladen.yaml 3ms (unchanged)
templates/definition/charger/delta.yaml 3ms (unchanged)
templates/definition/charger/demo-charger.yaml 5ms (unchanged)
templates/definition/charger/easee.yaml 4ms (unchanged)
templates/definition/charger/eebus.yaml 2ms (unchanged)
templates/definition/charger/elli-charger-connect.yaml 4ms (unchanged)
templates/definition/charger/elli-charger-pro.yaml 4ms (unchanged)
templates/definition/charger/em2go-home.yaml 2ms (unchanged)
templates/definition/charger/em2go.yaml 3ms (unchanged)
templates/definition/charger/etrel-duo.yaml 3ms (unchanged)
templates/definition/charger/etrel.yaml 2ms (unchanged)
templates/definition/charger/evbox-livo.yaml 2ms (unchanged)
templates/definition/charger/evse-din.yaml 2ms (unchanged)
templates/definition/charger/evsewifi.yaml 1ms (unchanged)
templates/definition/charger/fritzdect.yaml 3ms (unchanged)
templates/definition/charger/fronius-wattpilot.yaml 2ms (unchanged)
templates/definition/charger/go-e-v3.yaml 3ms (unchanged)
templates/definition/charger/go-e.yaml 2ms (unchanged)
templates/definition/charger/hardybarth-ecb1.yaml 3ms (unchanged)
templates/definition/charger/hardybarth-salia.yaml 2ms (unchanged)
templates/definition/charger/heidelberg.yaml 3ms (unchanged)
templates/definition/charger/hesotec.yaml 2ms (unchanged)
templates/definition/charger/homematic.yaml 5ms (unchanged)
templates/definition/charger/homewizard.yaml 1ms (unchanged)
templates/definition/charger/icharge-cion.yaml 2ms (unchanged)
templates/definition/charger/innogy-ebox.yaml 2ms (unchanged)
templates/definition/charger/keba-modbus.yaml 3ms (unchanged)
templates/definition/charger/keba-udp.yaml 8ms (unchanged)
templates/definition/charger/kse.yaml 5ms (unchanged)
templates/definition/charger/mennekes-compact.yaml 7ms (unchanged)
templates/definition/charger/mennekes-hcc3.yaml 3ms (unchanged)
templates/definition/charger/mystrom.yaml 2ms (unchanged)
templates/definition/charger/nrggen2.yaml 4ms (unchanged)
templates/definition/charger/nrgkick-bluetooth.yaml 2ms (unchanged)
templates/definition/charger/nrgkick-connect.yaml 2ms (unchanged)
templates/definition/charger/obo.yaml 3ms (unchanged)
templates/definition/charger/ocpp-abb.yaml 2ms (unchanged)
templates/definition/charger/ocpp-elvi.yaml 2ms (unchanged)
templates/definition/charger/ocpp-fronius-wattpilot.yaml 2ms (unchanged)
templates/definition/charger/ocpp-homecharge.yaml 2ms (unchanged)
templates/definition/charger/ocpp-orbis-viaris.yaml 1ms (unchanged)
templates/definition/charger/ocpp-pulsarplus.yaml 2ms (unchanged)
templates/definition/charger/ocpp.yaml 3ms (unchanged)
templates/definition/charger/openevse.yaml 2ms (unchanged)
templates/definition/charger/openwb-2.0.yaml 3ms (unchanged)
templates/definition/charger/openwb-pro.yaml 3ms (unchanged)
templates/definition/charger/openwb.yaml 2ms (unchanged)
templates/definition/charger/pantabox.yaml 1ms (unchanged)
templates/definition/charger/pcelectric-garo.yaml 2ms (unchanged)
templates/definition/charger/phoenix-charx.yaml 2ms (unchanged)
templates/definition/charger/phoenix-em-eth.yaml 1ms (unchanged)
templates/definition/charger/phoenix-ev-eth.yaml 3ms (unchanged)
templates/definition/charger/phoenix-ev-ser.yaml 1ms (unchanged)
templates/definition/charger/porsche-pmcc.yaml 2ms (unchanged)
templates/definition/charger/porsche-pmcp.yaml 2ms (unchanged)
templates/definition/charger/pracht-alpha.yaml 2ms (unchanged)
templates/definition/charger/pulsares.yaml 2ms (unchanged)
templates/definition/charger/pulsatrix.yaml 2ms (unchanged)
templates/definition/charger/scheider-evlink-v3.yaml 2ms (unchanged)
templates/definition/charger/shelly.yaml 2ms (unchanged)
templates/definition/charger/smaevcharger.yaml 3ms (unchanged)
templates/definition/charger/smartevse.yaml 2ms (unchanged)
templates/definition/charger/smartwb.yaml 1ms (unchanged)
templates/definition/charger/solax.yaml 3ms (unchanged)
templates/definition/charger/sungrow.yaml 2ms (unchanged)
templates/definition/charger/tapo.yaml 1ms (unchanged)
templates/definition/charger/tasmota.yaml 3ms (unchanged)
templates/definition/charger/tinkerforge-warp.yaml 3ms (unchanged)
templates/definition/charger/tinkerforge-warp3.yaml 2ms (unchanged)
templates/definition/charger/tplink.yaml 1ms (unchanged)
templates/definition/charger/twc3.yaml 2ms (unchanged)
templates/definition/charger/versicharge.yaml 2ms (unchanged)
templates/definition/charger/vestel.yaml 2ms (unchanged)
templates/definition/charger/victron-evcs.yaml 2ms (unchanged)
templates/definition/charger/victron.yaml 2ms (unchanged)
templates/definition/charger/wallbe-meter.yaml 2ms (unchanged)
templates/definition/charger/wallbe-pre2019-meter.yaml 2ms (unchanged)
templates/definition/charger/wallbe-pre2019.yaml 2ms (unchanged)
templates/definition/charger/wallbe.yaml 2ms (unchanged)
templates/definition/charger/webasto-next.yaml 2ms (unchanged)
templates/definition/charger/weidmüller.yaml 2ms (unchanged)
templates/definition/charger/zaptec.yaml 3ms (unchanged)
templates/definition/meter/abb-ab.yaml 2ms (unchanged)
templates/definition/meter/ac-elwa-2.yaml 2ms (unchanged)
templates/definition/meter/ac-elwa-e.yaml 2ms (unchanged)
templates/definition/meter/acrel-adw300.yaml 2ms (unchanged)
templates/definition/meter/alpha-ess-smile.yaml 3ms (unchanged)
templates/definition/meter/apsystems-ez1.yaml 3ms (unchanged)
templates/definition/meter/be-mpm3pm.yaml 1ms (unchanged)
templates/definition/meter/cfos.yaml 2ms (unchanged)
templates/definition/meter/cg-em24.yaml 2ms (unchanged)
templates/definition/meter/cg-emt1xx.yaml 2ms (unchanged)
templates/definition/meter/cg-emt3xx.yaml 3ms (unchanged)
templates/definition/meter/demo-battery.yaml 5ms (unchanged)
templates/definition/meter/demo-meter.yaml 4ms (unchanged)
templates/definition/meter/deye-hybrid-3p.yaml 4ms (unchanged)
templates/definition/meter/deye-hybrid-hp3.yaml 3ms (unchanged)
templates/definition/meter/deye-mi.yaml 2ms (unchanged)
templates/definition/meter/deye-storage.yaml 2ms (unchanged)
templates/definition/meter/deye-string.yaml 2ms (unchanged)
templates/definition/meter/discovergy.yaml 2ms (unchanged)
templates/definition/meter/dsmr.yaml 3ms (unchanged)
templates/definition/meter/dzg.yaml 2ms (unchanged)
templates/definition/meter/e3dc-modbus.yaml 2ms (unchanged)
templates/definition/meter/e3dc-rscp.yaml 3ms (unchanged)
templates/definition/meter/eastron-sdm120.yaml 2ms (unchanged)
templates/definition/meter/eastron-sdm220_230.yaml 3ms (unchanged)
templates/definition/meter/eastron-sdm72.yaml 2ms (unchanged)
templates/definition/meter/eastron-sdm72v2_630.yaml 3ms (unchanged)
templates/definition/meter/enphase.yaml 3ms (unchanged)
templates/definition/meter/fox-ess-h1.yaml 2ms (unchanged)
templates/definition/meter/fox-ess-h3.yaml 2ms (unchanged)
templates/definition/meter/fritzdect.yaml 2ms (unchanged)
templates/definition/meter/fronius-gen24.yaml 3ms (unchanged)
templates/definition/meter/fronius-ohmpilot.yaml 2ms (unchanged)
templates/definition/meter/fronius-solarapi-v1.yaml 3ms (unchanged)
templates/definition/meter/go-e-controller.yaml 2ms (unchanged)
templates/definition/meter/goodwe-dt.yaml 2ms (unchanged)
templates/definition/meter/goodwe-hybrid.yaml 3ms (unchanged)
templates/definition/meter/goodwe-wifi.yaml 3ms (unchanged)
templates/definition/meter/growatt-hybrid-tlxh.yaml 2ms (unchanged)
templates/definition/meter/growatt-hybrid.yaml 2ms (unchanged)
templates/definition/meter/homematic.yaml 4ms (unchanged)
templates/definition/meter/homewizard.yaml 2ms (unchanged)
templates/definition/meter/hoymiles-ahoydtu.yaml 2ms (unchanged)
templates/definition/meter/hoymiles-opendtu.yaml 2ms (unchanged)
templates/definition/meter/huawei-smartlogger.yaml 3ms (unchanged)
templates/definition/meter/huawei-sun2000-dongle-powersensor.yaml 3ms (unchanged)
templates/definition/meter/huawei-sun2000-dongle.yaml 3ms (unchanged)
templates/definition/meter/huawei-sun2000-rs485.yaml 2ms (unchanged)
templates/definition/meter/inepro.yaml 1ms (unchanged)
templates/definition/meter/janitza.yaml 2ms (unchanged)
templates/definition/meter/keba-kecontact.yaml 2ms (unchanged)
templates/definition/meter/kostal-ksem-inverter.yaml 2ms (unchanged)
templates/definition/meter/kostal-ksem.yaml 2ms (unchanged)
templates/definition/meter/kostal-piko-hybrid.yaml 2ms (unchanged)
templates/definition/meter/kostal-piko-legacy.yaml 2ms (unchanged)
templates/definition/meter/kostal-piko-mp-plus.yaml 2ms (unchanged)
templates/definition/meter/kostal-piko-pv.yaml 2ms (unchanged)
templates/definition/meter/kostal-plenticore.yaml 5ms (unchanged)
templates/definition/meter/lg-ess-home-8-10.yaml 3ms (unchanged)
templates/definition/meter/loxone.yaml 4ms (unchanged)
templates/definition/meter/mtec-eb-gen2.yaml 2ms (unchanged)
templates/definition/meter/mtec-eb-gen3.yaml 3ms (unchanged)
templates/definition/meter/mypv-wifi-meter.yaml 3ms (unchanged)
templates/definition/meter/mystrom.yaml 2ms (unchanged)
templates/definition/meter/openems.yaml 2ms (unchanged)
templates/definition/meter/orno.yaml 1ms (unchanged)
templates/definition/meter/p1monitor.yaml 2ms (unchanged)
templates/definition/meter/plexlog.yaml 3ms (unchanged)
templates/definition/meter/powerdog.yaml 2ms (unchanged)
templates/definition/meter/powerfox-poweropti.yaml 3ms (unchanged)
templates/definition/meter/rct-power.yaml 2ms (unchanged)
templates/definition/meter/saj-h2.yaml 2ms (unchanged)
templates/definition/meter/sax.yaml 2ms (unchanged)
templates/definition/meter/sbc-axx3.yaml 2ms (unchanged)
templates/definition/meter/schneider-iem3000.yaml 2ms (unchanged)
templates/definition/meter/senec-home.yaml 2ms (unchanged)
templates/definition/meter/senergy.yaml 2ms (unchanged)
templates/definition/meter/shelly-1pm.yaml 2ms (unchanged)
templates/definition/meter/shelly-3em.yaml 2ms (unchanged)
templates/definition/meter/shelly-pro-3em.yaml 2ms (unchanged)
templates/definition/meter/siemens-7kt1665.yaml 2ms (unchanged)
templates/definition/meter/siemens-pac2200.yaml 2ms (unchanged)
templates/definition/meter/slimmelezer-v2.yaml 2ms (unchanged)
templates/definition/meter/slimmelezer.yaml 4ms (unchanged)
templates/definition/meter/sma-datamanager.yaml 3ms (unchanged)
templates/definition/meter/sma-energymeter.yaml 2ms (unchanged)
templates/definition/meter/sma-homemanager.yaml 5ms (unchanged)
templates/definition/meter/sma-hybrid.yaml 3ms (unchanged)
templates/definition/meter/sma-inverter-modbus.yaml 5ms (unchanged)
templates/definition/meter/sma-inverter-speedwire.yaml 2ms (unchanged)
templates/definition/meter/sma-sbs-modbus.yaml 4ms (unchanged)
templates/definition/meter/sma-si-modbus.yaml 3ms (unchanged)
templates/definition/meter/sma-webbox.yaml 3ms (unchanged)
templates/definition/meter/smartfox-em2.yaml 2ms (unchanged)
templates/definition/meter/smartfox.yaml 2ms (unchanged)
templates/definition/meter/sofarsolar-g3.yaml 5ms (unchanged)
templates/definition/meter/sofarsolar.yaml 3ms (unchanged)
templates/definition/meter/solaranzeige-mqtt.yaml 2ms (unchanged)
templates/definition/meter/solaredge-hybrid.yaml 3ms (unchanged)
templates/definition/meter/solaredge-inverter.yaml 3ms (unchanged)
templates/definition/meter/solarlog.yaml 2ms (unchanged)
templates/definition/meter/solarman.yaml 3ms (unchanged)
templates/definition/meter/solarmax-inverter-smt.yaml 2ms (unchanged)
templates/definition/meter/solarmax-maxstorage.yaml 3ms (unchanged)
templates/definition/meter/solarwatt-myreserve-matrix.yaml 3ms (unchanged)
templates/definition/meter/solarwatt.yaml 2ms (unchanged)
templates/definition/meter/solax-hybrid-cloud.yaml 4ms (unchanged)
templates/definition/meter/solax-inverter-cloud.yaml 3ms (unchanged)
templates/definition/meter/solax.yaml 3ms (unchanged)
templates/definition/meter/solis-hybrid-s.yaml 2ms (unchanged)
templates/definition/meter/solis-hybrid.yaml 3ms (unchanged)
templates/definition/meter/solis.yaml 3ms (unchanged)
templates/definition/meter/sonnenbatterie_eco56.yaml 4ms (unchanged)
templates/definition/meter/sonnenbatterie.yaml 8ms (unchanged)
templates/definition/meter/storaxe.yaml 2ms (unchanged)
templates/definition/meter/sungrow-hybrid.yaml 4ms (unchanged)
templates/definition/meter/sungrow-inverter.yaml 3ms (unchanged)
templates/definition/meter/sunspec-battery-control.yaml 3ms (unchanged)
templates/definition/meter/sunspec-hybrid.yaml 3ms (unchanged)
templates/definition/meter/sunspec-inverter-control.yaml 4ms (unchanged)
templates/definition/meter/sunspec-inverter.yaml 2ms (unchanged)
templates/definition/meter/tapo.yaml 2ms (unchanged)
templates/definition/meter/tasmota-3p.yaml 4ms (unchanged)
templates/definition/meter/tasmota-sml.yaml 3ms (unchanged)
templates/definition/meter/tasmota.yaml 3ms (unchanged)
templates/definition/meter/tesla-powerwall.yaml 4ms (unchanged)
templates/definition/meter/thor.yaml 1ms (unchanged)
templates/definition/meter/tibber-pulse.yaml 2ms (unchanged)
templates/definition/meter/tplink.yaml 2ms (unchanged)
templates/definition/meter/tq-em.yaml 2ms (unchanged)
templates/definition/meter/tq-em420.yaml 2ms (unchanged)
templates/definition/meter/varta.yaml 2ms (unchanged)
templates/definition/meter/victron-energy.yaml 4ms (unchanged)
templates/definition/meter/volkszaehler-http.yaml 3ms (unchanged)
templates/definition/meter/volkszaehler-importexport.yaml 3ms (unchanged)
templates/definition/meter/volkszaehler-ws.yaml 2ms (unchanged)
templates/definition/meter/vzlogger.yaml 2ms (unchanged)
templates/definition/meter/youless.yaml 2ms (unchanged)
templates/definition/tariff/allinpower.yaml 2ms (unchanged)
templates/definition/tariff/amber.yaml 2ms (unchanged)
templates/definition/tariff/awattar.yaml 2ms (unchanged)
templates/definition/tariff/electricitymaps.yaml 3ms (unchanged)
templates/definition/tariff/elering.yaml 3ms (unchanged)
templates/definition/tariff/energinet.yaml 2ms (unchanged)
templates/definition/tariff/energy-charts-api.yaml 3ms (unchanged)
templates/definition/tariff/entsoe.yaml 3ms (unchanged)
templates/definition/tariff/groupe-e.yaml 2ms (unchanged)
templates/definition/tariff/gruenstromindex.yaml 2ms (unchanged)
templates/definition/tariff/ngeso.yaml 2ms (unchanged)
templates/definition/tariff/octopus-api.yaml 2ms (unchanged)
templates/definition/tariff/octopus-productcode.yaml 3ms (unchanged)
templates/definition/tariff/pun.yaml 2ms (unchanged)
templates/definition/tariff/smartenergy.yaml 1ms (unchanged)
templates/definition/tariff/tibber.yaml 2ms (unchanged)
templates/definition/vehicle/aiways.yaml 2ms (unchanged)
templates/definition/vehicle/audi.yaml 2ms (unchanged)
templates/definition/vehicle/bmw.yaml 2ms (unchanged)
templates/definition/vehicle/carwings.yaml 2ms (unchanged)
templates/definition/vehicle/citroen.yaml 5ms (unchanged)
templates/definition/vehicle/dacia.yaml 2ms (unchanged)
templates/definition/vehicle/ds.yaml 3ms (unchanged)
templates/definition/vehicle/evnotify.yaml 3ms (unchanged)
templates/definition/vehicle/fiat.yaml 2ms (unchanged)
templates/definition/vehicle/flobz.yaml 4ms (unchanged)
templates/definition/vehicle/ford-connect.yaml 4ms (unchanged)
templates/definition/vehicle/ford.yaml 3ms (unchanged)
templates/definition/vehicle/hyundai.yaml 2ms (unchanged)
templates/definition/vehicle/iso15118.yaml 3ms (unchanged)
templates/definition/vehicle/jaguar-landrover.yaml 1ms (unchanged)
templates/definition/vehicle/kia.yaml 2ms (unchanged)
templates/definition/vehicle/mazda2mqtt.yaml 4ms (unchanged)
templates/definition/vehicle/mercedes.yaml 4ms (unchanged)
templates/definition/vehicle/mg.yaml 2ms (unchanged)
templates/definition/vehicle/mg2mqtt.yaml 3ms (unchanged)
templates/definition/vehicle/mini.yaml 2ms (unchanged)
templates/definition/vehicle/mz2mqtt.yaml 4ms (unchanged)
templates/definition/vehicle/nissan-ariya.yaml 3ms (unchanged)
templates/definition/vehicle/nissan.yaml 2ms (unchanged)
templates/definition/vehicle/niu-e-scooter.yaml 4ms (unchanged)
templates/definition/vehicle/offline.yaml 3ms (unchanged)
templates/definition/vehicle/opel.yaml 3ms (unchanged)
templates/definition/vehicle/ovms.yaml 3ms (unchanged)
templates/definition/vehicle/peugeot.yaml 3ms (unchanged)
templates/definition/vehicle/polestar.yaml 1ms (unchanged)
templates/definition/vehicle/porsche.yaml 1ms (unchanged)
templates/definition/vehicle/renault.yaml 2ms (unchanged)
templates/definition/vehicle/seat-cupra.yaml 2ms (unchanged)
templates/definition/vehicle/seat.yaml 1ms (unchanged)
templates/definition/vehicle/skoda-enyaq.yaml 2ms (unchanged)
templates/definition/vehicle/skoda.yaml 2ms (unchanged)
templates/definition/vehicle/smart-hello.yaml 4ms (unchanged)
templates/definition/vehicle/smart.yaml 1ms (unchanged)
templates/definition/vehicle/tesla.yaml 8ms (unchanged)
templates/definition/vehicle/teslalogger.yaml 8ms (unchanged)
templates/definition/vehicle/teslamate.yaml 3ms (unchanged)
templates/definition/vehicle/tronity.yaml 5ms (unchanged)
templates/definition/vehicle/volvo-connected.yaml 2ms (unchanged)
templates/definition/vehicle/volvo.yaml 2ms (unchanged)
templates/definition/vehicle/vw.yaml 2ms (unchanged)
templates/definition/vehicle/zero.yaml 2ms (unchanged)
tests/basics.evcc.yaml 5ms (unchanged)
tests/battery-settings.evcc.yaml 6ms (unchanged)
tests/config-empty.evcc.yaml 4ms (unchanged)
tests/config-grid-only.evcc.yaml 3ms (unchanged)
tests/config-with-tariffs.evcc.yaml 5ms (unchanged)
tests/config-with-vehicle.evcc.yaml 3ms (unchanged)
tests/fatal-db.evcc.yaml 2ms (unchanged)
tests/fatal-syntax.evcc.yaml 1ms (unchanged)
tests/heating.evcc.yaml 5ms (unchanged)
tests/plan.evcc.yaml 7ms (unchanged)
tests/simulator.evcc.yaml 7ms (unchanged)
tests/smart-cost-only.evcc.yaml 3ms (unchanged)
tests/statistics.evcc.yaml 4ms (unchanged)
tests/vehicle-error.evcc.yaml 4ms (unchanged)
util/templates/defaults.yaml 47ms (unchanged)
```

Reply:
Ok, I'll have to recheck our eslint config then. But please introduce a new vehicle instead of reusing a parameter.

Reply:
That's [no-param-reassign](https://eslint.org/docs/latest/rules/no-param-reassign). 👍

Reply:
If you want, I can add this rule in another PR and fix the bugs.

Reply:
Yes please.